src/components/DropdownFilter.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div>
+    <v-select
+      :placeholder="placeholder"
+      :options="options"
+      :reduce="options => options.slug"
+      v-model="selection"
+      label="name"
+    ></v-select>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+:deep(.vs__selected + .vs__search) {
+  display: none;
+}
+:deep(.v-select) {
+  .vs__selected {
+    padding: 5px 10px;
+  }
+  &.vs--open.vs--single {
+    .vs__dropdown-menu {
+      margin: 0;
+    }
+    .vs__selected {
+      position: static;
+    }
+  }
+  .vs__fade-enter-active,
+  .vs__fade-leave-active {
+    &.vs__dropdown-menu {
+      margin: 0;
+    }
+  }
+  .vs__search {
+    background: none;
+    border: none;
+    &:focus {
+      border: none;
+      font-size: inherit;
+      padding: 6px 12px;
+    }
+  }
+  .vs__search {
+    font-size: 14px;
+    padding: 6px 12px;
+  }
+}
+:deep(button) {
```

Comment:
Can we build on the PicnicSCSS and then use `%extend` (pseudocode) directives to really get all the styles we need?  Or even, within a scoped block can we import Picnic?  It's small...

Reply:
https://github.com/ua-snap/fishbiz-map/issues/20